Makefile

```diff
@@ -81,8 +88,12 @@ configgen: requirements .configgen
 	echo "" >> conf/st2.conf.sample
 	. $(VIRTUALENV_DIR)/bin/activate; python ./tools/config_gen.py >> conf/st2.conf.sample;
 
+.PHONY: .install-pika
+.install-pika:
+	$(VIRTUALENV_DIR)/bin/pip install --upgrade pika
```

Comment:
Why are we installing pika when we have kombu?

Reply:
The `tools/direct_queue_publisher.py` [imports `pika`](https://github.com/StackStorm/st2/blob/9a560d2d5b6a354f8b3a405b48722a54f47458c0/tools/direct_queue_publisher.py#L21), and that was messing up the pylint check.

Reply:
We don't even use that tool. Maybe switch that tool to use kombu?

Reply:
If we don't use that tool we should just remove it entirely. I'm also not entirely sure why any of my changes caused pylint to check (and then complain) about this file when it wasn't doing it before, but ¯\\\_(ツ)\_/¯.

I added this quick hack because without it the Travis tests were failing due to pylint complaining that `pika` wasn't installed.